src/context/values.test.ts

```diff
@@ -16,6 +16,16 @@ describe("ContextValue functions", () => {
     sandbox.restore();
   });
 
+  it("setContextValue() should correctly context value and execute the setContext command", async () => {
+    const key = ContextValues.resourceSelectedForCompare;
+    const value = true;
+
+    await setContextValue(key, value);
+
+    assert.ok(executeCommandStub.calledWith("setContext", key, value));
+    assert.deepStrictEqual(getContextValue<boolean>(key), value);
+  });
+
```

Comment:
Unrelated to this PR; brought over from https://github.com/confluentinc/vscode/pull/847